app/Http/Controllers/Admin/UserController.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Models\User;
+use Throwable;
+use App\Http\Requests\UserRequest;
+
+class UserController extends Controller
+{
+
+    public function index()
+    {
+        $users = User::Alluser()->get();
```

Comment:
tương tự dưới đây nữa o_O
```suggestion
        $users = User::allUser()->get();
```